### PR TITLE
Generate unique default names for smart cell variables

### PR DESCRIPTION
**Pull request by jonatanklosko**
https://user-images.githubusercontent.com/17034772/159354243-ca904a8c-d549-4e84-980d-b60b55ff86c8.mp4

### CODE REVIEW

File: lib/kino/counter.ex [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    :ets.update_counter(@table_name, key, {2, 1}, {key, 0})
+  end
+
+  @doc """
+  Bumps the counter to `value` unless it already has a higher value.
+
+  Returns the current counter value.
+  """
+  @spec bump(term(), integer()) :: integer()
+  def bump(key, value) do
```
Comment (josevalim) [inline lib/kino/counter.ex:29]: `bump` typically means "bump by value". Maybe set or put would be better here? :)

**Review by josevalim — APPROVED** [review]
Beautiful, just one nitpick!

File: lib/kino/counter.ex (hunk 1 above)
Comment (jonatanklosko) [reply lib/kino/counter.ex:29]: `set` and `put` sound like they would always override the value, while in this case we only do it if it increases :thinking:

Comment (josevalim) [reply lib/kino/counter.ex:29]: @jonatanklosko `at_least`, `update_max`, `put_max`, all work. Your call. :)

Comment (josevalim) [reply lib/kino/counter.ex:29]: I mean they all work _in addition_ to what you suggested. :)

**Pull request merged**